### ISSUE TITLE
Data type for LinkPropPredDataset(name=name, root="datasets", preprocess=True)

### ISSUE THREAD

**Issue by JuliaGast** [JuliaGast/TGB2#2]
LinkPropPredDataset 
Currently the datatypes for numpy dataset are all `float`
We need `int` for the following:

- [ ] sources
- [ ] destinations
- [ ] timestamps
- [ ] edge_idxs

**Comment by shenyangHuang** [comment]
Yes, these can be converted before hand, can be improved in the next version

**Comment by shenyangHuang** [comment]
issue addressed in #8

**Comment by JuliaGast** [comment]
thanks. closing this.
